platform/admin/ui/src/main/webapp/js/views/configuration/ConfigurationEdit.view.js

```diff
@@ -255,9 +257,29 @@ define([
         /**
          * Submit to the backend.
          */
-        submitData: function() {
+        submitData: function () {
+            var spinner = new Spinner(define({
```

Comment:
Might be nice if we could reuse the spinnerConfig.js here and just change the color or whatever else you need to here.